archlinux/content.md

```diff
@@ -0,0 +1,32 @@
+# What is Arch Linux?
+
+Arch Linux, is a lightweight and flexible Linux® distribution that tries to Keep It Simple.
+
+Currently we have official packages optimized for the x86-64 architecture. We complement our official package sets with a community-operated package repository that grows in size and quality each and every day.
+
+Our strong community is diverse and helpful, and we pride ourselves on the range of skillsets and uses for Arch that stem from it. Please check out our forums and mailing lists to get your feet wet. Also glance through our wiki if you want to learn more about Arch.
+
+%%LOGO%%
+
+# About this image
+
+The base filesystem tarball for this image is auto-generated in Arch Linux infrastructure at the beginning of each month. Given the rolling-release nature of Arch Linux, images are tagged with the timestamp of the date they were generated. For example, `%%IMAGE`:20181107 was generated the Seventh of November of 2018. The latest tag will contain the latest build always.
+
+This image is intended to serve the following goals:
+
+	- Provide the Arch experience in a Docker Image
+	- Provide the most simple but complete image to base every other upon
+	- pacman needs to work out of the box
+	- All installed packages have to be kept unmodified
+
+## Base packages
+
+Given that this image doesn't intend to contain an init system, several packages of the `base` group have been removed.
```

Comment:
Is the list of packages removed static enough that they could reasonably be listed here?  If not, is there perhaps somewhere in the GitHub repository we could link for users curious which ones this includes? (perhaps https://github.com/archlinux/archlinux-docker/pull/6 or https://github.com/archlinux/archlinux-docker/blob/master/packages)

Looking at that `packages` file, it does seem to contain the `systemd` package (and I understand why it would probably have to at this point), but it does seem like the wording here is slightly misleading as a result. :sweat_smile:

Reply:
Right! I'll review this and make sure it's up to date. I'll also add a forward reference to the packages list.